content/docs/try-react.md

```diff
@@ -0,0 +1,37 @@
+---
+id: try-react
+title: Try React
+permalink: docs/try-react.html
+next: add-react-to-a-new-app.html
+redirect_from:
+  - "docs/installation.html"
+  - "download.html"
+  - "downloads.html"
+  - "docs/tooling-integration.html"
+  - "docs/package-management.html"
+  - "docs/language-tooling.html"
+  - "docs/environments.html"
+---
+
+Try React online or set up your local dev environment
+
+## Online
+
+If you're just interested in playing around with React, you can use an online code playground. Try a Hello World template on [CodePen](codepen://hello-world) or [CodeSandbox](https://codesandbox.io/s).
+
+## HTML Template
+
+If you prefer to use your own text editor, you can also [download this HTML file](https://raw.githubusercontent.com/reactjs/reactjs.org/master/static/html/single-file-example.html), edit it, and open it from the local filesystem in your browser. It does a slow runtime code transformation, so don't use it in production.
+
+## Next Steps
+
+### Quick Start
+
+Head over to the [Quick Start](/docs/hello-world.html) section to try some examples.
```

Comment:
I think it's a bit misleading to call QS section that (since it doesn't have that many useful "examples").

Maybe it would be better to direct people either to Tutorial (for those looking for a hands-on practical introduction) or to QS (but frame it as a step-by-step introduction to React concepts).

Reply:
Linking to the Tutorial is a great idea.